openmc/nuclide.py

```diff
@@ -37,3 +44,19 @@ def __new__(cls, name):
     @property
     def name(self):
         return self
+
+    @property
+    def element_name(self):
+        return ATOMIC_SYMBOL[zam(self)[0]]
+
+    @property
+    def protons(self):
+        return ATOMIC_NUMBER[self.element_name]
+
+    @property
+    def nucleons(self):
+        return zam(self)[1]
+
+    @property
+    def neutrons(self):
+        return zam(self)[1]-zam(self)[0]
```

Comment:
to reduce duplication, would this be better as
```
return self.nucleons() - self.protons()
```

Reply:
Thanks for the comment boss. I am just exploring the nuclide.py and elements.py classes. I am also thinking about trying to make the element enrichment a little more generic so that it works for any element not just Uranium. Perhaps that is a task for another day.